### DISCUSSION TITLE
download workflow form responses from my slack by API

### DISCUSSION THREAD

**Discussion by enimath** [OAI/OpenAPI-Specification#2782] [Q&A]
Hello,

I am trying to download workflow form responses from my slack by API but there is no info in API about it.

Is that possible ?
if Yes can You please put example how to do it ?

BR

**Comment by chenjinzhi1990** [comment]
![image](https://user-images.githubusercontent.com/25029027/186376636-1d6916e8-cae5-49f0-b9de-164a2762fb76.png)

**Comment by handrews** [comment]
Closing as off-topic (and yet also answered!)
